src/main/docs/guide/toc.yml

```diff
@@ -189,6 +189,9 @@ security:
     logout: Logout Controller
     refresh: Refresh Controller
   securityEvents: Security Events
+spring:
+  title: Micronaut And Spring
+  micronautBeans: Adding Micronaut Beans To A Spring Context
```

Comment:
I don't think we need a whole new top level topic on this. Could you move this to the "Bridging Spring AOP section" or as a subsection of the DI section?

Reply:
https://github.com/micronaut-projects/micronaut-core/commit/5b619de6ab176391a3932779084a4236dead33b1